libzpc.spec

```diff
@@ -0,0 +1,86 @@
+Name:		libzpc
+Version:	1.0.0
+Release:	1%{?dist}
+Summary:	Open Source library for the IBM Z Protected-key crypto feature
+
+License:	MIT License
+Url:		https://github.com/opencryptoki/libzpc
+Source0:	%{url}/archive/v%{version}/%{name}-%{version}.tar.gz
+
+ExclusiveArch:	s390x
+BuildRequires:	cmake gcc g++ make
```

Comment:
please put each BR on its own line, it's easier to track changes

Reply:
Done.
And I added
```
BuildRequires:	json-c-devel
BuildRequires:	glibc-devel
```
The lib also needs libpthread, which is in glibc-devel.

Reply:
The compiler brings in its C library, so BR: glibc-devel is redundant, please remove.

Reply:
Done.